site/layouts/partials/docs-navbar.html

```diff
@@ -42,8 +42,41 @@
             <div class="bd-search" id="docsearch" data-bd-docs-version="{{ .Site.Params.docs_version }}"></div>
           </li>
           {{ end }}
+
           {{ partial "docs-versions" . }}
         </ul>
+        <ul class="navbar-nav flex-row">
+          <li class="nav-item dropdown d-flex">
+            <a href="#" class="nav-link dropdown-toggle gap-1 mx-0 bg-transparent" id="bd-theme" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static">
```

Comment:
Should be a button right ?

Reply:
This might be to tackle outside of this PR because I used the same thing as for the versions.  But I agree that it should be a `<button>` rather than a `<a>`!